src/libs/PolicyUtils.ts

```diff
@@ -197,7 +202,7 @@ function getPolicyBrickRoadIndicatorStatus(policy: OnyxEntry<Policy>, isConnecti
 }
 
 function getPolicyRole(policy: OnyxInputOrEntry<Policy> | SearchPolicy, currentUserLogin: string | undefined) {
-    return policy?.role ?? policy?.employeeList?.[currentUserLogin ?? '-1']?.role;
+    return policy?.role ?? (currentUserLogin ? policy?.employeeList?.[currentUserLogin]?.role : undefined);
```

Comment:
I think it's best to revert these lint fixes, since they're not errors we introduced. I think we're good to ignore those errors for now and fix in a followup as it's bunching too many changes into this one PR that's fixing something else. @grgia Do you agree?